demos/motortwit/motortwit/generate_data.py

```diff
@@ -0,0 +1,95 @@
+import asyncio
+import random
+
+from bson import ObjectId
+from faker import Factory
+
+from motortwit import db
+from motortwit.main import init_mongo
+from motortwit.security import generate_password_hash
+
+
+conf = {"host": "127.0.0.1",
+        "port": 8080,
+        "mongo": {
+            "host": "127.0.0.1",
+            "port": 27017,
+            "database": "motortwit",
+            "max_pool_size": 1}
+        }
+
+
+# TODO: validate using trafarets before insert
+async def insert_data(collection, values):
+    await collection.insert(values)
+    return values
+
+
+async def generate_users(mongo, schema, rows, fake):
+    values = []
+    pw_hash = generate_password_hash('123456')
```

Comment:
May be this magic number should come from config?

Reply:
this is dummy password for each generated user, not read need to store in config.